### PR TITLE
Docs(web): Fix Sass implementation notes

### PR DESCRIPTION
**Pull request by adamkudrna**
Fix minor documentation issues introduced in #1260.

### CODE REVIEW

File: packages/web-react/src/components/Modal/README.md [hunk 1]
```diff
@@ -318,8 +318,8 @@ content is longer than user's viewport.
 #### ⚠️ DEPRECATION NOTICE
 
 The `isScrollable` prop will be set to `false` by default in the next major release and the ModalDialog will be made
-non-scrollable by default. It will be possible to re-enable the inside scrolling by setting the
-`isScrollable` boolean prop.
+non-scrollable by default. It will be possible to re-enable the inside scrolling by setting the `isScrollable` prop to
+`true` explicitly.
```
Comment (literat) [inline packages/web-react/src/components/Modal/README.md:322]: In React the `true` is not needed to be set.

File: packages/web-twig/src/Resources/components/Modal/README.md [hunk 2]
```diff
@@ -323,8 +323,8 @@ content is longer than user's viewport.
 #### ⚠️ DEPRECATION NOTICE
 
 The `isScrollable` prop will be set to `false` by default in the next major release and the ModalDialog will be made
-non-scrollable by default. It will be possible to re-enable the inside scrolling by setting the
-`isScrollable` prop to `true` explicitly.
+non-scrollable by default. It will be possible to re-enable the inside scrolling by setting the `isScrollable` prop to
+`true` explicitly.
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Modal/README.md:327]: Also in  the web-twig.

**Comment by adamkudrna** [comment]
As I was passing by, I also fixed some `aria-controls` ID references that incorrectly started with `#`.

**Review by literat — APPROVED** [review]
Looks nice and shiny 👍

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**